qemu/tests/cfg/multi_nics_verify.cfg

```diff
@@ -2,12 +2,20 @@
     virt_test_type = qemu
     type = multi_nics_verify
     no RHEL.3.9
-    nics_num = 27
     start_vm = no
     login_timeout = 1800
+    variants:
+        - nic_virtio:
+            only virtio_net
+            nics_num = 27
+        - nic_e1000e:
+            # Limited by e1000e device and seabios firmware
+            only e1000e
+            nics_num = 8
```

Comment:
Is this limitation only for PCIE(q35)? If so, suggest:
```python
q35:
    nics_num = 8
```

By the way, other architectures do not support e1000e device.

Reply:
Yes, e1000e is only used under Q35 machines.But this problem is limited by seabios firmware, it will not appear if we use ovmf.  I tested if it was not for Q35 machines, it would not run to e1000e device.

Reply:
Okay, then please add `only x86_64` here.

Reply:
Hi, @PaulYuuu, I have added it. Please help me review it again, thanks.